portal/static/portal/sass/partials/_forms.scss

```diff
@@ -70,14 +70,10 @@ form {
     }
   }
 
-  h1 {
-    color: $color-text-secondary;
-    text-align: center;
-  }
-
+  h1,
   p {
     color: $color-text-secondary;
-    text-align: left;
+    text-align: center;
```

Comment:
I think this breaks the style changes in the student register form. We wanted the centered text to be left aligned so that it is more readable.